File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/quests/instances/Pulverization.java
package it.paranoidsquirrels.idleguildmaster.storage.data.quests.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.quests.Quest;
import it.paranoidsquirrels.idleguildmaster.storage.data.quests.QuestsManager;

/* JADX INFO: loaded from: classes3.dex */
public class Pulverization extends Quest {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.quests.Quest
    public void calculateTargetProgress(int i) {
        this.targetProgress = 1000L;
    }

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.quests.Quest
    protected void configure() {
        this.idName = R.string.quest_pulverization_name;
        this.idDescription = R.string.quest_pulverization_description;
        this.defaultRarity = 3;
        this.minimumDifficulty = 7;
    }

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.quests.Quest
    public void realignStaticReference() {
        QuestsManager.pulverization = this;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/quests/instances/Pulverization.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.quests.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.quests.Quest
import it.paranoidsquirrels.idleguildmaster.storage.data.quests.QuestsManager

class Pulverization : Quest() {
    override fun calculateTargetProgress(difficulty: Int) {
        targetProgress = 1000L
    }

    override fun configure() {
        idName = R.string.quest_pulverization_name
        idDescription = R.string.quest_pulverization_description
        defaultRarity = 3
        minimumDifficulty = 7
    }

    override fun realignStaticReference() {
        QuestsManager.pulverization = this
    }
}
